.pre-commit-config.yaml

```diff
@@ -24,3 +24,7 @@ repos:
     rev: 'v0.0.269'
     hooks:
       - id: ruff
+  - repo: https://github.com/google/yamlfmt
+    rev: v0.14.0
```

Comment:
Is there a reason to use v0.14.0 instead of [v0.17.0](https://github.com/google/yamlfmt/releases)?

Reply:
Nope! That was taken from the pre-commit docs from yamlfmt. I've just sent a PR to them just to update those docs :+1: